cmd/http-server/startcmd/start.go

```diff
@@ -159,23 +159,26 @@ func VueHandler(publicDir string, opts *ariesJSOpts, trustblocAgentOpts *trustbl
 
 		// aries js opts
 		if urlPath == ariesStartupOptsPath {
-			j, _ := json.Marshal(opts) // nolint errcheck
+			j, _ := json.Marshal(opts) // nolint:errcheck // don't know why err is ignored
+
+			w.Write(j) // nolint:errcheck,gosec // don't know why err is ignored
 
-			w.Write(j) // nolint errcheck
 			return
 		}
 
 		// trustbloc agent js opts
 		if urlPath == trustblocStartupOptsPath {
-			j, _ := json.Marshal(trustblocAgentOpts) // nolint errcheck
+			j, _ := json.Marshal(trustblocAgentOpts) // nolint:errcheck // don't know why err is ignored
+
+			w.Write(j) // nolint:errcheck,gosec // don't know why err is ignored
```

Comment:
Rather than adding a "don't know..." type of comment, it would be better to create a follow-up issue to look at it and then link it here.

Reply:
@DRK3 PR #368 refactored this part and is removing `VueHandler`. We should refactor/do followup there.